src/Enum/ErrorMessage.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Bootstrap5\Enum;
+
+/**
+ * Is an enumeration of error messages.
+ */
+enum ErrorMessage: string
+{
+    case TAG_NOT_EMPTY_STRING = 'Tag cannot be empty string.';
```

Comment:
If it seems like too much for now, because it is the first widget, I would like to keep it until the end of the whole refactoring, and see the final result, this guarantees that the exceptions will always have the same message everywhere.